### PR TITLE
DriverAvailability: driverAvailability index page

### LINKED ISSUE #17: DriverAvailability: Index Page

### PR DESCRIPTION
**Pull request by wzqd**
In this pr, I added index page for Driver Availability and completed tests.

Snipaste_2024-03-07_02-42-46

close #17

### CODE REVIEW

**Review by cydthe — DISMISSED** [review]
LGTM

**Review by Rytuose — APPROVED** [review]
LGTM

**Review by Rytuose — DISMISSED** [review]
LGTM

**Review by calee14 — DISMISSED** [review]
LGTM

File: frontend/src/tests/pages/DriverAvailability/DriverAvailabilityIndexPage.test.js [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+    //                 <DriverAvailabilityIndexPage />
+    //             </MemoryRouter>
+    //         </QueryClientProvider>
+    //     );
+
+    //     await waitFor(() => { expect(screen.getByTestId(`${testId}-cell-row-0-col-id`)).toHaveTextContent("2"); });
+    //     expect(screen.getByTestId(`${testId}-cell-row-1-col-id`)).toHaveTextContent("3");
+    //     expect(screen.getByTestId(`${testId}-cell-row-2-col-id`)).toHaveTextContent("4");
+
+    // });
```
Comment (andrewpengucsb) [inline frontend/src/tests/pages/DriverAvailability/DriverAvailabilityIndexPage.test.js:99]: Remove commented test

Comment (andrewpengucsb) [reply frontend/src/tests/pages/DriverAvailability/DriverAvailabilityIndexPage.test.js:99]: Other than that good to go, ping me when you fix

**Review by andrewpengucsb — CHANGES_REQUESTED** [review]

File: frontend/src/tests/pages/DriverAvailability/DriverAvailabilityIndexPage.test.js (hunk 1 above)
Comment (wzqd) [reply frontend/src/tests/pages/DriverAvailability/DriverAvailabilityIndexPage.test.js]: I have removed the commented tests. thx for the review

**Review by andrewpengucsb — APPROVED** [review]

**Review by Rytuose — APPROVED** [review]
LGTM

**Pull request merged**